src/program/librapay.js

```diff
@@ -0,0 +1,234 @@
+// @flow
+
+import {
+  Account,
+  PublicKey,
+  Connection,
+  Transaction,
+  SystemProgram,
+} from '@solana/web3.js';
+import path from 'path';
+import fs from 'mz/fs';
+import * as lo from 'buffer-layout';
+
+import {newSystemAccountWithAirdrop} from '../util/new-system-account-with-airdrop';
+import * as InstructionData from './instruction_data';
+import {sendAndConfirmTransaction} from '../util/send-and-confirm-transaction';
+import {MoveLoader} from './move-loader';
+
+const sizeOfGenesisAccount = 5176;
```

Comment:
Comment where 5176 come from?  Or is that obvious for folks who know about Libra?

Reply:
It's not obvious and may change in the future.  On the rust side we calculate it, on the js side we just go with what we know now